### DISCUSSION TITLE
有没有可能做一个局域网内同步播放的功能？

### DISCUSSION THREAD

**Discussion by XingZhe-Li** [lyswhut/lx-music-desktop#1006] [Ideas]
网易QQ都出了一起听功能，想着洛雪有没有可能做一个。
感觉实现起来应该不容易，Github上现在还找不到多少有这个功能的项目。
仅作为一个提议。

**Comment by lyswhut** [comment]
局域网的话可以做，说说具体的想法？

**Reply by XingZhe-Li** [reply]
几台设备间共享播放列表与播放进度？描述起来貌似就是这样，不过这个几大于2的话就得考虑采用中心化还是去中心化的问题了。

**Reply by lyswhut** [reply]
一般的局域网下彼此的距离都会很近，家庭网络下更是在同一个屋里，局域网共享播放进度感觉意义不是很大？
觉得控制播放暂停切歌等功能可能更实用一些

**Reply by XingZhe-Li** [reply]
对于“局域网”来说，是的。但是如果考虑ipv6的话就不一定了，不过目前ipv6用户还太少，功能开发出来用得上的人不多。果然还是控制播放更实用一些。

**Reply by Gavin-OP** [reply]
想要这个功能+1，就类似spotify，电脑在播放音乐的时候，可以用手机打开spotify在床上切歌，搜索，控制音量等等等等
